webapp/i18n/en.json

```diff
@@ -52,6 +52,7 @@
   "GalleryCard.delete": "Delete",
   "GalleryCard.duplicate": "Duplicate",
   "GroupBy.ungroup": "Ungroup",
+  "General.BoardCount": "{count, plural, =0 {# Boards} one {# Board} other {# Boards}}",
```

Comment:
Probably you can simplify this as `"{count, plural, one {# Board} other {# Boards}}"`

Reply:
Done.

Reply:
done.